queries.sql

```diff
@@ -7,4 +7,75 @@ SELECT date_of_birth from animals WHERE name  IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT name from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+/* Start transaction query for species column */
+
+BEGIN;
+  UPDATE animals SET species = 'unspecified';
+
+  select * from animals;
+
+ROLLBACK TRANSACTION;
+
+/* end transaction query for species column */
+
+/* Start transaction query for updating species column for animals that have name ending in mon and those that do not have species already set */
+
+BEGIN;
+
+    UPDATE animals SET species = 'digimon' WHERE name LIKE 'mon%';
+
+    UPDATE animals SET species = 'pokemon' WHERE species = '';
+
+    /* Confirm updates */
+
+    SELECT * FROM animals;
+
+COMMIT WORK;
```

Comment:
- [ ] Just like I mentioned above, it will be nice to also verify your changes hereafter committing.
Please add: `SELECT species from animals;` after the commit statement.